app/lib/frontend/handlers/package.dart

```diff
@@ -95,8 +95,16 @@ Future<shelf.Response> packageVersionsListHandler(
   final isAdmin =
       await packageBackend.isPackageAdmin(package, userSessionData?.userId);
 
+  final bool isLiked = (userSessionData == null)
+      ? false
+      : await accountBackend.getPackageLikeStatus(
```

Comment:
It seems to be unnecessary to lookup `User` just to use its already known id to lookup the like status. Let's refactor `getPackageLikeStatus` to use only the id?